website/docs/getting-started.mdx

```diff
@@ -0,0 +1,455 @@
+![Application detail view showing events for an object](/img/dashboard-application-events.png)
+
+**Reconciliation Graph tab**
+![Application detail view showing reconciliation graph - a directional graph showing object relationships](/img/dashboard-application-reconciliation.png)
+
+#### Source details view
+Finally lets look at the Source in more detail, click `GitRepository/flux-system` from the summary at the top of the page.
+
+![Source detail view showing details for an object](/img/dashboard-source-flux.png)
+
+As with an Application detail view, you can see key information about how the resource is defined. Then beneath alongside the Events tab, is a Related Automations view. This shows all the `kustomization` objects which have this object as their Source.
+
+
+## Part 2 - Deploying and viewing podinfo application
+
+Now that you have a feel for how to navigate the dashboard. Let's deploy a new application and explore that as well. In this section we will use the [podinfo](https://github.com/stefanprodan/podinfo) sample web application.
+
+### Deploying podinfo
+
+1. Clone or navigate back to your git repository where you have bootstrapped Flux, for example:
+
+```
+git clone https://github.com/$GITHUB_USER/fleet-infra
+cd fleet-infra
+```
+
+2. Create a `GitRepository` Source for podinfo
+
+```
+flux create source git podinfo \
+  --url=https://github.com/stefanprodan/podinfo \
+  --branch=master \
+  --interval=30s \
+  --export > ./clusters/my-cluster/podinfo-source.yaml
+```
+
+3. Commit and push the `podinfo-source` to the `fleet-infra` repository
+
+```
+git add -A && git commit -m "Add podinfo source"
+git push
+```
+
+4. Create a `kustomization` to build and apply the podinfo manifest
+
+```
+flux create kustomization podinfo \
+  --target-namespace=flux-system \
+  --source=podinfo \
+  --path="./kustomize" \
+  --prune=true \
+  --interval=5m \
+  --export > ./clusters/my-cluster/podinfo-kustomization.yaml
+```
+
+5. Commit and push the `podinfo-kustomization` to the `fleet-infra` repository
+
+```
+git add -A && git commit -m "Add podinfo kustomization"
+git push
+```
+
+### View the application in Weave GitOps
+
+Flux will detect the updated `fleet-infra` and add podinfo. If we navigate back to the [dashboard](http://localhost:9001/) you should see the podinfo application appear.
+
+![Applications summary view showing Flux System, Weave GitOps and Podinfo](/img/dashboard-applications-with-podinfo.png)
+
+Click on podinfo and you will see details about the deployment, including that there are 2 replicas available.
+
+![Applications details view for podinfo showing 2 pods](/img/dashboard-podinfo-details.png)
+
+### Customize podinfo
+
+To customize a deployment from a repository you don’t control, you can use Flux in-line patches. The following example shows how to use in-line patches to change the podinfo deployment.
+
+1. Add the following to the field spec of your podinfo-kustomization.yaml file:
+
+```
+patches:
```

Comment:
This line should have 2 leading spaces, so this block can just be copy-pasted into the file. The rest of the block is already appropriately indented.